### ISSUE TITLE
How to change Video type mp4

### ISSUE THREAD

**Issue by prakash50166** [fbaiodias/react-video-recorder#56]
Hi i have configure react-video-recorder in my project but i can't play recording video in safari browser can you please give me the solution or how to change from video file webm to mp4 in react-video-recorder npm package

**Comment by VapiSoft** [comment]
Hi prakash50166,
I am facing the same problem, because webm cann't be displayed on iPhone.
I can do the conversion on to mp4 on my server (using ffmpeg) but it takes a lot of time.
So if you find a solution in the client, let me know.
Thanks.

**Comment by GiancarloZ** [comment]
Hi, I'm encountering the same issue, Has anyone found a solution?

**Comment by ephron-systems** [comment]
I'm also facing this problem, Please give the solutions

**Comment by jayna456** [comment]
Did anyone find any solution?

**Comment by LeonidIvanov** [comment]
Same thing. We can't play recorded video.

**Comment by asadrazashah** [comment]
Most browsers support webm for recording , so it is not possible to convert into mp4 on frontend you have to use a tool ffmpeg to convert it into mp4 using backend server .
But its a time taking task even for videos with small size

**Comment by kamal-choudhary** [comment]
facing same problem

**Comment by kamal-choudhary** [comment]
I am converting the RECORDED video to mp4 like this on frontend:

```
 onRecordingComplete={(videoBlob) => {
            const newFile = new File([videoBlob], 'video-title.mp4', {
              type: 'video/mp4'
            })
}}
```

working for me in Chrome.

**Comment by najathi** [comment]
@kamal-choudhary 

The methods you mentioned are creating a new File or Blob object and setting the MIME type to 'video/mp4'. However, this doesn't convert the video from WebM to MP4 format, it just changes the file type. If you try to play this 'mp4' file, most likely it won't work because the internal data is still WebM, not MP4.

**Comment by LetMeDream** [comment]
@kamal-choudhary

Yep, that solution of yours still wouldn't work on Safari, which is the issue I am currently facing.
